### ISSUE TITLE
ProgressBar and ProgressRing controls are not working in SDK 1.1 preview 3

### ISSUE THREAD

**Issue by hassaanmasood** [microsoft/microsoft-ui-xaml#7164]
### Describe the bug

If I include ProgressBar or ProgressRing control on the Window, the app simple does not load. And also the error is not meaningful.

I used the simple XAML:
`<ProgressBar Width="130" Value="0" />`

Error:

Unhandled exception at 0x5BE8BAC9 (Microsoft.ui.xaml.dll) in App3.exe: 0xC000027B: An application-internal exception has occurred (parameters: 0x122D8960, 0x00000002).

### Steps to reproduce the bug

1. Create a new Blank App in WinUI 3 C# with SDK 1.1 Preview 3. 
2. Add a ProgressRing or ProgressBar control on the MainWindow.
3. Run the app

### Expected behavior

The app should start normally when ProgressBar or ProgressRing control is added.

### Screenshots

Can't add screenshot as the app is not starting.

### NuGet package version

WinUI 3 - Windows App SDK 1.1 Preview 3: 1.1.0-preview3

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (21H2): Build 22000

**Comment by castorix** [comment]
They work fine for me (Windows 10 21H1) =>

![ProgressBar_ProgressRing](https://user-images.githubusercontent.com/22345506/171266428-e66ed41a-de97-4b3e-aa9e-57db56de1b33.gif)

**Comment by ghost1372** [comment]
can you share your xaml? i think you removed grid, put progressbar in grid
```
<Grid>
<Progressbar/>
</Grid>
```

**Comment by hassaanmasood** [comment]
@castorix and @ghost1372 Guys I think you are not considering that I am using WinAppSDK 1.1 preview 3. (Not the original 1.0). In 1.0 it was working fine. But with 1.1 preview 3 its not working. And I am on Windows 11 22000 Build. 

@ghost1372 I am not using any specific xaml. I wanted to create a simple app and it required progress ring as the very first control. But it did not work. I haven't added any logic yet to the code. You can just try creating a blank app with SDK 1.1 preview 3 and simply add a progress bar or ring.

**Comment by castorix** [comment]
> But with 1.1 preview 3 its not working.

I also tested with 1.1 preview 3

**Comment by ghost1372** [comment]
> > But with 1.1 preview 3 its not working.
> 
> I also tested with 1.1 preview 3

put Progressbar in Window

**Comment by castorix** [comment]
> > > But with 1.1 preview 3 its not working.
> > 
> > 
> > I also tested with 1.1 preview 3
> 
> put Progressbar in Window

You can see in the GIF that I have put a ProgressRing and a ProgressBar...

```
        <ProgressBar Width="130" IsIndeterminate="True" ShowPaused="False" ShowError="False" />
        <ProgressRing IsActive="True" />
```

**Comment by castorix** [comment]
> > > But with 1.1 preview 3 its not working.
> > 
> > 
> > I also tested with 1.1 preview 3
> 
> put Progressbar in Window

I have put it in a standard StackPanel

**Comment by hassaanmasood** [comment]
Below is the simple XAML of MainWindow.xaml. And I also tried using VS 2019 on Windows 10. Its not working. This is definitely an issue with 1.1 preview 3 SDK. 

```
<Window
    x:Class="App1.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:App1"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">

    <Grid>
        <ProgressRing IsActive="True" />
    </Grid>
</Window>
```

**Comment by hassaanmasood** [comment]
@StephenLPeters can you please ask someone to look into this and if there is a temporary workaround.

**Comment by krschau** [comment]
We've noted internally that this worked with v1.0.3 but does not work in 1.1.0 and 1.1.1.

**Comment by asklar** [comment]
RNW will be affected by this so adding the tag. We should look into prioritizing this, CC @krschau

**Comment by kmahone** [comment]
@hassaanmasood thanks for reporting this issue. The problem is understood and I have a fix available internally.

This issue is hit whenever a ProgressRing, ProgressBar (or a few other controls) are created when Windows.UI.dll is not already loaded into the process. So it is possible to work-around this issue by forcing Windows.UI.dll to be loaded before your page/wndow loads. I was able to do this by adding a line of code to my window's ctor:

```
        public MainWindow()
        {
            var coreDispatcher = this.Dispatcher;
            this.InitializeComponent();
        }
```

Even though this line of code doesn't really do anything, it does have the effect of getting Windows.UI.dll loaded into the process and so you no longer hit a problem with ProgressRing or ProgressBar.

**Comment by gabbybilka** [comment]
This issue was fixed and released in Windows App SDK 1.1.3 - check out the [release notes](https://aka.ms/windowsappsdk/1.1/1.1.3/release-notes) and [NuGet](https://aka.ms/windowsappsdk/1.1/1.1.3) to try it out 🎉

**Comment by GuildOfCalamity** [comment]
Has anyone experience the issue with the progress bar inserting a 1 pixel line through the middle?
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/81915293/a9e371b8-f371-4eaa-92a0-8ecfb375b4b2)
What causes this? I have set the `BorderThickness` to 0 and `BorderBrush` to Transparent.
